defaults/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 remi_repo_url: "http://rpms.remirepo.net/enterprise/remi-release-{{ ansible_distribution_major_version }}.rpm"
 remi_repo_gpg_key_url: "http://rpms.remirepo.net/RPM-GPG-KEY-remi"
+remi_validate_certs: true
```

Comment:
can you call this `remi_repo_validate_certs`? With that change I'm happy to merge it. Please also add a note about it in the README file so people will discover the var there.